### ISSUE TITLE
Firefox webdriver support maybe is broken

### ISSUE THREAD

**Issue by vimagick** [horejsek/python-webdriverwrapper#10]
- selenium (3.0.1)
- webdriverwrapper (2.3.0)

I'm tring to [fill_out][1] form, but the result of `get_elem()` is `selenium.webdriver.firefox.webelement.FirefoxWebElement` instead of `webdriverwrapper.forms.Form`.

I need to override the `driver._web_element_cls` (FirefoxWebElement => WebElement) to make it work.

```python
from webdriverwrapper import Firefox
from selenium.webdriver.remote.webelement import WebElement

driver = Firefox()
driver._web_element_cls = WebElement
driver.get('https://www.macys.com/account/signin')
driver.get_elm('signInForm').fill_out({
    'email': 'username@gmail.com',
    'password': 'password'
})
```

[wrapper.py][2] cannot handle `FirefoxWebElement` very well.

```diff
    @classmethod
    def _convert_result(cls, res):
        if type(res) is WebElement: # <-- FirefoxWebElement is not WebElement
            res = cls._convert_into_webelementwrapper(res)
        elif isinstance(res, (list, tuple)):
            for index, item in enumerate(res):
                res[index] = cls._convert_result(item)
        return res
```

[1]: http://horejsek.github.io/python-webdriverwrapper/#webdriverwrapper.forms.Form.fill_out
[2]: https://github.com/horejsek/python-webdriverwrapper/blob/master/webdriverwrapper/wrapper.py#L60

**Comment by horejsek** [comment]
Thanks for report. Last months I work on server-side apps so I don't have to use it and guys told me it works fine with Chrome. Probably no one tried Firefox with upgraded Selenium. I will look into it. :-)

**Comment by vimagick** [comment]
@horejsek
Yes. Chrome works fine. :+1:

**Comment by horejsek** [comment]
I found that also there is problem with Python 3 in some cases. I fixed that first. I did also some fixes for this issue (Firefox and Selenium 3) but there are still few tests which has to be solved. I will finish it (hopefully) in next week.

**Comment by horejsek** [comment]
Should be fixed in version 2.4.0. If there will be still some problem, let me know.
